test/unit/region.build-region.spec.js

```diff
@@ -258,10 +258,7 @@ describe('Region', function() {
       });
 
       it('throws an error', function() {
-        expect(this.buildRegion).to.throw(Marionette.Error, new Marionette.Error({
-          message: 'Improper region configuration type.',
-          url: 'marionette.region.html#region-configuration-types'
```

Comment:
these errors are strange, probably a side-effect of whether we're using Metal or Marionette Error and how it's exported. the two things to look for are are URL and version, which were issues before but should be resolved